src/lib.rs

```diff
@@ -1,41 +1,385 @@
 #![no_std]
 
-use gstd::{debug, msg, prelude::*};
+use gear_lib::non_fungible_token::token::{TokenId, TokenMetadata};
+use gstd::{async_main, exec, msg, prelude::*, ActorId};
+use nft_pixelboard_io::*;
 
-#[no_mangle]
-pub unsafe extern "C" fn handle() {
-    debug!("handle()");
-    let payload = String::from_utf8(msg::load_bytes()).expect("Invalid handle message");
+mod utils;
+use utils::*;
```

Comment:
It's more idiomatic to point the module when calling extern functions: https://doc.rust-lang.org/stable/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html
```suggestion
```